utils/wrk/post.lua

```diff
@@ -1,4 +1,4 @@
 wrk.method = "POST"
 wrk.headers["Authorization"] = "Basic <base_64_encoded_writekey_colon>"
 wrk.headers["Content-Type"] = "application/json"
-wrk.body = '{"batch": [{"channel":"android-sdk","destination_props":{"AF":{"af_uid":"1566363489499-3377330514807116178"}},"context":{"app":{"build":"1","name":"RudderAndroidClient","namespace":"com.rudderlabs.android.sdk","version":"1.0"},"device":{"id":"49e4bdd1c280bc00","manufacturer":"Google","model":"Android SDK built for x86","name":"generic_x86"},"locale":"en-US","network":{"carrier":"Android"},"screen":{"density":420,"height":1794,"width":1080},"traits":{},"user_agent":"Dalvik/2.1.0 (Linux; U; Android 9; Android SDK built for x86 Build/PSR1.180720.075)"},"event":"Demo Track - New","integrations":{"All":true,"Google Analytics":true},"messageId":"1566904375469-1baf3108-647a-4f20-9867-3072056a07f5","properties":{"label":"Demo Label","category":"Demo Category","value":5},"originalTimestamp":"2019-08-12T05:08:30.909Z","sentAt":"2019-08-12T05:08:30.909Z"}]}'
+wrk.body = '{"batch":[{"channel":"android-sdk","context":{"app":{"build":"1","name":"RudderAndroidClient","namespace":"com.rudderlabs.android.sdk","version":"1.0"},"device":{"id":"49e4bdd1c280bc00","manufacturer":"Google","model":"Android SDK built for x86","name":"generic_x86"},"locale":"en-US","network":{"carrier":"Android"},"screen":{"density":420,"height":1794,"width":1080},"library":{"name":"com.rudderstack.android.sdk.core"},"traits":{},"user_agent":"Dalvik/2.1.0 (Linux; U; Android 9; Android SDK built for x86 Build/PSR1.180720.075)"},"event":"Demo Track - New","type":"track","integrations":{"All":false,"Amplitude":true},"messageId":"1566904375469-1baf3108-647a-4f20-9867-3072056a07f5","properties":{"label":"Demo Label","category":"Demo Category","value":5},"originalTimestamp":"2019-08-12T05:08:30.909Z","sentAt":"2019-08-12T05:08:30.909Z"}]}'
```

Comment:
Same here. Just remove this key or make default as All: true

Reply:
done